website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-be-chart.md

```diff
@@ -136,7 +136,7 @@ To ensure that the Business Edition Helm chart runs, you need to make some chang
      APPSMITH_KEYCLOAK_DB_NAME: "keycloak"
    ```
 
-## Install v2 Helm chart 
+## Install v3 Helm chart
```

Comment:
What about the installations which are already on v2?

Reply:
@pratapaprasanna - Please confirm.

Reply:
- This will also work and we can avoid the confusion around using a specific version
- We have a new version v3 which fixes few issues

Reply:
Does this mean that the updates to the command will always pull the latest version, and should the heading be v3 in that case?

Reply:
Yes, that makes more sense rather than confusing people with versions we can just remove it